Documentation/supported-mod-list.md

```diff
@@ -18,324 +21,328 @@
         - [Singleplayer mods with one map or single maps](#singleplayer-mods-with-one-map-or-single-maps-1)
    - [List of XashXT-based mods](#list-of-xashxt-based-mods)
    - [List of maps for They Hunger](#list-of-maps-for-they-hunger)
-   - [List of games and mods with custom gamedll](#list-of-games-and-mods-with-custom-gamedll)
-   - [List of mods for Counter Strike](#list-of-mods-for-counter-strike)
+   - [List of They Hunger-based mods](#list-of-they-hunger-based-mods)
```

Comment:
What is the difference between They Hunger-based mods and maps for They Hunger?

Reply:
Single maps and mod that based our hlsdk fork.